### PR TITLE
Depreciate old documentation

### PR DESCRIPTION
**Pull request by haixuanTao**
This PR removes the old documentation in favor of the newest one dora.carsmos.ai

It also refactor our README.md

### CODE REVIEW

File: examples/python-operator-dataflow/run.sh [hunk 1]
```diff
@@ -4,6 +4,7 @@ python3 -m venv .env
 . $(pwd)/.env/bin/activate
 # Dev dependencies
 pip install maturin
+pip install patchelf
```
Comment (haixuanTao) [inline examples/python-operator-dataflow/run.sh:7]: This addition just fix some linking issue that can happen when people tries the python operator example

File: README.md [hunk 2]
```diff
@@ -2,176 +2,103 @@
...
 
-`dora` primary support is with `Linux` as it is the primary OS for both Cloud and small computers. If you wish to use `dora` with another OS, please compile from source.
+## Why dora-rs?
 
----
-## 📖 Documentation
+In 2023, AI is booming! Robotic framework however hasn't changed much in years... This is why we create dora-rs! dora-rs is a new robotic framework that brings modernity into robotic application.
 
-The documentation can be found here: [https://dora-rs.github.io/dora/](https://dora-rs.github.io/dora/) 
+dora-rs can already show impressive performance, up to 17x speedup compared to ROS2! This is the result of using our own shared memory server and Apache Arrow to achieve zero copy!
```
Comment (phil-opp) [inline README.md:40]: This speedup is for dora Python compared to ROS2 Python, right? So maybe we should mention Python here?

**Review by phil-opp — APPROVED** [review]
Thanks!

File: README.md (hunk 2 above)
Comment (haixuanTao) [reply README.md]: Yeah absolutely!

**Comment by phil-opp** [comment]
I removed the `deploy` job from the list of required status checks, as it was removed by this PR.

**Pull request merged**